common/types.go

```diff
@@ -168,3 +169,71 @@ type LogConfig struct {
 	FileCount   uint64
 	TimeTracker bool
 }
+
+type Block struct {
+	Id         string
+	StartIndex int64
+	EndIndex   int64
+	Size       int64
+}
+
+// list that holds blocks containing ids and corresponding offsets
+type BlockOffsetList struct {
+	BlockOffsetList []*Block //blockId to offset mapping
+	SmallFile       bool     // does it consist of blocks
```

Comment:
If block list is empty then its implicit that file is not blocked, do we need this flag seperatly?

Reply:
I'll look into removing this